std_msgs/Header header 
float64 N
float64 E
float64 dN
float64 dE
float64 yaw
float64 yaw_rate
float64 vx